src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/transformer/ReferralTransformers.kt

```diff
@@ -46,20 +43,3 @@ fun ReferralUpdate.toApi() = ApiReferralUpdate(
   oasysConfirmed = oasysConfirmed,
   hasReviewedProgrammeHistory = hasReviewedProgrammeHistory,
 )
-
-fun List<ReferralSummaryProjection>.toApi(): List<ApiReferralSummary> {
```

Comment:
Feel like this has snuck in from another commit - it's probably a pain to rebase, so would you mind at least adding a line to explain why this has gone?

Reply:
Yeah, rebasing went a bit sideways because there were so many changes to hook in. Not sure why this has been left over, but I'll add a brief line to the commit.